### ISSUE TITLE
[dv/tools] Mark tests that fail to avoid coverage

### ISSUE THREAD

**Issue by matutem** [lowRISC/opentitan#7102]
Do not use coverage metrics from failing tests.
In VCS we use $cm_post when the test ends, using the UVM warning/error/fatal flags.
We need to add support for other tools.

**Comment by msfschaffner** [comment]
@sriyerg is this still relevant?

**Comment by sriyerg** [comment]
Yes it is, but we have a workaround, so low priority (ICEBOX maybe).

**Comment by msfschaffner** [comment]
Ack - re-labelled accordingly.
